src/main/java/org/beanpod/switchboard/dto/mapper/EncoderMapper.java

```diff
@@ -16,6 +16,7 @@
     nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
 public interface EncoderMapper {
 
+  @Mapping(target = "output", qualifiedByName = "toModelsShallow")
```

Comment:
Shouldn't it be toModelShallow without the 's'?

Reply:
No, because the output is a list, so I have to specify a mapper from collections.

Reply:
No, because the output is a list, so I have to specify a mapper from collections. Also I think the conversion needs to be from a list to a set and mapstruct doesn't like that.

Reply:
Gotcha!